backend/coins/eth/types/types.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2018 Shift Devices AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package types
+
+// EthereumTransaction holds information specific to Ethereum.
+type EthereumTransaction interface {
```

Comment:
I think I would prefer to rename the other transactions we have from `wrappedTransaction` to `gethTransaction` and the one in `etherscan` maybe to `EtherScanTransaction`. This `Transaction` interface should then simply extend `coin.Transaction`.

Reply:
it is already etherscan.Transaction. By go convention, `etherscan.EtherScanTransaction` should be avoided because it stutters.

No opinion on `gethTransaction`, but renaming it is out of scope.

This transaction doesn't need to extend `coin.Transaction` (obviously, as the code compiles and works without it).